### PR TITLE
Update Accept-CH format

### PR DESCRIPTION
**Pull request by rowan-m**
Updates https://web.dev/user-agent-client-hints to cover the changes in [Intent to Remove: Rename User-Agent Client Hint ACCEPT-CH tokens](https://groups.google.com/a/chromium.org/g/blink-dev/c/t-S9nnos9qU/m/pUFJb00jBAAJ). This should not be merged until the intent is fully confirmed, but this is ready for review.

- Added a "gotcha" at the top to alert developers of the change
- Updated references to the previous `Accept-CH` format
- Simplified the table now the response / request values match
- Updated the reference for Feature Policy
- Remove the inline demo, as hints won't be delegated without a specific Feature Policy

### CODE REVIEW

File: src/site/content/en/blog/user-agent-client-hints/index.md [hunk 1]
```diff
@@ -131,35 +143,35 @@ The server can choose to vary its responses, for example by serving images at an
...
+prefix that can be specified via the `Accept-CH` server response header. For all
+the details, start with [the
 explainer](https://github.com/WICG/ua-client-hints/blob/master/README.md) and
 then dive into the [full proposal](https://wicg.github.io/ua-client-hints/).
 
 {% Aside %}
-  Client Hints are **only sent over secure connections**, so make sure you have
-  [migrated your site to HTTPS](https://web.dev/why-https-matters/).
+Client Hints are **only sent over secure connections**, so make sure you have
+[migrated your site to HTTPS](https://web.dev/why-https-matters/).
```
Comment (mihajlija) [inline src/site/content/en/blog/user-agent-client-hints/index.md:160]:
```suggestion
[migrated your site to HTTPS](/why-https-matters).
```

File: src/site/content/en/blog/user-agent-client-hints/index.md [hunk 2]
```diff
@@ -131,35 +143,35 @@ The server can choose to vary its responses, for example by serving images at an
...
 The new set of hints is available from Chromium 84, so let's explore how it
 all works.
 
-## User-Agent Client Hints in Chromium 84
+## User-Agent Client Hints from Chromium 84
 
 User-Agent Client Hints will only be enabled gradually on Chrome Stable as
 [compatibility
 concerns](https://bugs.chromium.org/p/chromium/issues/detail?id=1091285) are
 resolved. To force the functionality on for testing, ensure you:
```
Comment (mihajlija) [inline src/site/content/en/blog/user-agent-client-hints/index.md:171]:
```suggestion
resolved. To force the functionality on for testing:
```

File: src/site/content/en/blog/user-agent-client-hints/index.md [hunk 3]
```diff
@@ -131,35 +143,35 @@ The server can choose to vary its responses, for example by serving images at an
...
 
-  * are on Chrome 84 **beta** or equivalent
-  * enable the `chrome://flags/#enable-experimental-web-platform-features` flag
+- are on Chrome 84 **beta** or equivalent
+- enable the `chrome://flags/#enable-experimental-web-platform-features` flag
```
Comment (mihajlija) [inline src/site/content/en/blog/user-agent-client-hints/index.md:174]:
```suggestion
- Use Chrome 84 **beta** or equivalent.
- Enable the `chrome://flags/#enable-experimental-web-platform-features` flag.
```

File: src/site/content/en/blog/user-agent-client-hints/index.md [hunk 4]
```diff
@@ -131,35 +143,35 @@ The server can choose to vary its responses, for example by serving images at an
 appropriate resolution.
 
 {% Aside %}
-  There are ongoing discussions on enabling Client Hints on an initial request,
-  but you should be considering [responsive
-  design](/responsive-web-design-basics) or progressive enhancement before going
-  down this route.
+There are ongoing discussions on enabling Client Hints on an initial request,
+but you should be considering [responsive
```
Comment (mihajlija) [inline src/site/content/en/blog/user-agent-client-hints/index.md:147]:
```suggestion
but you should consider [responsive
```

File: src/site/content/en/blog/user-agent-client-hints/index.md [hunk 5]
```diff
@@ -30,6 +30,18 @@ first step to eventually reducing User-Agent string granularity.
...
+The header format for User-Agent Client Hints is changing so the `Accept-CH`
+tokens exactly match the returned headers. That means where a site would
+previously have sent `Accept-CH: UA-Platform` to receive the
+`Sec-CH-UA-Platform` header, now the site sends `Accept-CH: Sec-CH-UA-Platform`.
+If you are already using User-Agent Client Hints then you should send both
+formats until the change has fully rolled out in stable Chromium. See [Intent to
+Remove: Rename User-Agent Client Hint ACCEPT-CH
+tokens](https://groups.google.com/a/chromium.org/g/blink-dev/c/t-S9nnos9qU/m/pUFJb00jBAAJ)
+for updates.
+{% endAside %}
```
Comment (mihajlija) [inline src/site/content/en/blog/user-agent-client-hints/index.md:43]:
```suggestion
{% Banner 'caution', 'body' %}
If you are already using User-Agent Client Hints, pay attention to the upcoming changes.
The header format is changing so the `Accept-CH` tokens exactly match the 
returned headers. Previously a site could have sent `Accept-CH: UA-Platform` to receive the
`Sec-CH-UA-Platform` header and now that site should send `Accept-CH: Sec-CH-UA-Platform`.
If you've already implemented User-Agent Client Hints, send both
formats until the change has fully rolled out in stable Chromium. See [Intent to
Remove: Rename User-Agent Client Hint ACCEPT-CH
tokens](https://groups.google.com/a/chromium.org/g/blink-dev/c/t-S9nnos9qU/m/pUFJb00jBAAJ)
for updates.
{% endBanner %}
```

Comment (mihajlija) [reply src/site/content/en/blog/user-agent-client-hints/index.md:43]: I've suggested a slight rephrase here that identifies the target audience for this info upfront and switched to a banner component to make it more prominent.

**Comment by kaycebasques** [comment]
@rowan-m & @mihajlija what's the current status here?

**Comment by rowan-m** [comment]
@kaycebasques thanks for the reminder! I need to address @mihajlija's feedback, but I also want to hold off publishing until the blink change gets its 3 lgtms.

**Comment by rowan-m** [comment]
Reminder: set the `updated` value in the article once the change is ready to go.

**Comment by rowan-m** [comment]
LGTMs received on blink-dev thread, we're good to go.

**Comment by rowan-m** [comment]
@kaycebasques this is all good to go whenever you've got time for a quick once over.

**Review by mihajlija — APPROVED** [review]

**Pull request merged**